src/event-notion/index.ts

```diff
@@ -324,91 +325,138 @@ export const pingForTAPandCSIDeadlines = async (
   databaseId: string,
   config: EventNotionPipelineConfig,
 ) => {
-  // The day for which we ping for any non-submitted events.
-  // This is just basically to look for events that are 23 days away from now.
-  //
   // Ideally, do ALL network queries in one and filter later.
-  const firstDayToPingForTODO = DateTime.now().plus({ days: 23 });
-  const secondDayToPingForTODO = DateTime.now().plus({ days: 22 });
-  const thirdDayToPingForTODO = DateTime.now().plus({ days: 21 });
-  const firstDayToPingForInProgress = DateTime.now().plus({ days: 16 });
-  const secondDayToPingForInProgress = DateTime.now().plus({ days: 15 });
-  const thirdDayToPingForInProgress = DateTime.now().plus({ days: 14 });
+  var dateTimeNow = DateTime.now();
```

Comment:
Code quality: don't use `var` (it has weird behaviors with blocks and stuff). Let's use `const` and `let` instead.